scvi/data/fields/_layer_field.py

```diff
@@ -101,7 +114,10 @@ def transfer_field(
         return self.register_field(adata_target)
 
     def get_summary_stats(self, state_registry: dict) -> dict:
-        return state_registry.copy()
+        return {self.count_stat_key: state_registry[self.N_VARS_KEY]}
```

Comment:
the variable name `count_stat_key` doesn't make much sense to me, what's going on here?

Reply:
it just names the summary stat based on the registry key. E.g. n_batch